### PR TITLE
Feature/business rating

### PR DESCRIPTION
**Pull request by Orelongz**
#### What does this PR do?
It provides an avenue to rate a business through its reviews

#### Description of Task to be completed?
image upload

#### What are the relevant Pivotal Tracker stories?
Pivotal Tracker Id: #157535726

### CODE REVIEW

File: client/src/components/pages/Rating.jsx [hunk 1]
```diff
@@ -0,0 +1,77 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import shortid from 'shortid';
+
+const propTypes = {
+  rating: PropTypes.number
+}
+class Rating extends Component {
```
Comment (Daramola98) [inline client/src/components/pages/Rating.jsx:8]: Good work trying to write your own component for ratings, but there seems to be an issue as the stars don't clear out after review has been submitted, you can try to get the component to re-render after review has been submitted or check out the react package REACT STARS which can be found here https://www.npmjs.com/package/react-stars

File: client/src/actions/businessAction.js [hunk 2]
```diff
@@ -100,12 +101,26 @@ const userBusinesses = () => (dispatch) => {
     });
 };
 
+const gottenBusinessRating = rating => ({
+  type: BUSINESS_RATING,
+  rating
+});
+
+const businessRating = businessId => (dispatch) => {
```
Comment (clintfidel) [inline client/src/actions/businessAction.js]: change your variable name.

File: server/src/controllers/reviewController.js [hunk 3]
```diff
@@ -76,6 +78,42 @@ export default class ReviewController {
...
+   */
+  static getBusinessRating(req, res) {
+    const { businessId } = req.params;
+
+    const isNotUUID = checkUUID(res, businessId, 'Business');
+    if (isNotUUID) return isNotUUID;
+
+    return Review.findAll({ where: { businessId } })
+      .then((reviews) => {
+        if (reviews === null) {
```
Comment (clintfidel) [inline server/src/controllers/reviewController.js:96]: why are you doing this check?, I don't think you understand your code. This seems like you don't understand what this code does.

File: client/src/components/pages/Rating.jsx (hunk 1 above)
Comment (Orelongz) [reply client/src/components/pages/Rating.jsx:8]: Thanks @Daramola98 for the feedback. Would get back to you on this.

File: client/src/actions/businessAction.js [hunk 4]
```diff
@@ -100,12 +101,26 @@ const userBusinesses = () => (dispatch) => {
     });
 };
 
+const gottenBusinessRating = rating => ({
```
Comment (clintfidel) [inline client/src/actions/businessAction.js:104]: I don't seem to get what this means.

**Pull request merged**